replica.yml

```diff
@@ -24,6 +32,27 @@ services:
     volumes:
       - geth:/geth
       - ./kustomize/replica/bases/configmaps/:/scripts/
+    <<: *logging
+    labels:
+      - traefik.enable=true
```

Comment:
Definitely don't want to add this to the default replica.yaml.

Reply:
Moved out